third_party/android/android_configure.bzl

```diff
@@ -45,18 +45,21 @@ def _android_autoconf_impl(repository_ctx):
     ndk_home = repository_ctx.os.environ.get(_ANDROID_NDK_HOME)
     ndk_api_level = repository_ctx.os.environ.get(_ANDROID_NDK_API_VERSION)
 
-    sdk_rule = "pass"
+    sdk_rule = ""
     if all([sdk_home, sdk_api_level, build_tools_version]):
         sdk_rule = _ANDROID_SDK_REPO_TEMPLATE % (
             sdk_home,
             sdk_api_level,
             build_tools_version,
         )
 
-    ndk_rule = "pass"
+    ndk_rule = ""
     if all([ndk_home, ndk_api_level]):
         ndk_rule = _ANDROID_NDK_REPO_TEMPLATE % (ndk_home, ndk_api_level)
 
+    if ndk_rule == "" and sdk_rule == "":
+        sdk_rule = "pass"
```

Comment:
Actually why is this changed?

Reply:
If both SDK and NDK are missing, I decided to add just pass statement instead of nothing.
I guess it might not be necessary, and empty function would work too